packages/webapp/src/components/CardWithStatus/TaskCard/styles.module.scss

```diff
@@ -88,3 +88,12 @@
   color: var(--red700);
   border-bottom: 1px solid var(--red700);
 }
+
+.inactive {
+  color: var(--grey500);
+}
+
+.firstInitial.inactive {
+  background-color: var(--grey500);
+  color: white;
+}
```

Comment:
@Duncan-Brain I don't have access to figma but maybe someone can confirm the correct color for the inactive label on the task card view.